
From: Hanna Linder <hannal@us.ibm.com>

Another simple patch to complete the /i386 conversion to pci_get_device.  I
was able to compile and boot this patch to verify it didn't break anything
(on my T22).

Signed-off-by: Hanna Linder <hannal@us.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/pci/acpi.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/i386/pci/acpi.c~changed-pci_find_device-to-pci_get_device arch/i386/pci/acpi.c
--- 25/arch/i386/pci/acpi.c~changed-pci_find_device-to-pci_get_device	2004-10-05 02:01:43.691159360 -0700
+++ 25-akpm/arch/i386/pci/acpi.c	2004-10-05 02:01:43.694158904 -0700
@@ -41,7 +41,7 @@ static int __init pci_acpi_init(void)
 		printk(KERN_INFO "** was specified.  If this was required to make a driver work,\n");
 		printk(KERN_INFO "** please email the output of \"lspci\" to bjorn.helgaas@hp.com\n");
 		printk(KERN_INFO "** so I can fix the driver.\n");
-		while ((dev = pci_find_device(PCI_ANY_ID, PCI_ANY_ID, dev)) != NULL)
+		while ((dev = pci_get_device(PCI_ANY_ID, PCI_ANY_ID, dev)) != NULL)
 			acpi_pci_irq_enable(dev);
 	} else {
 		printk(KERN_INFO "** PCI interrupts are no longer routed automatically.  If this\n");
_
